include-build/roborazzi-core/build.gradle

```diff
@@ -59,10 +59,10 @@ kotlin {
         def composeBom = platform('androidx.compose:compose-bom:2022.12.00')
```

Comment:
Platform BOMs can be in the version catalog too (and should be IMO).

Reply:
If the following changes are made, a build error will occur. If I can't resolve it, I won't change it.

```diff
diff --git a/include-build/roborazzi-core/build.gradle b/include-build/roborazzi-core/build.gradle
index 6eee9fe..d7161c3 100644
--- a/include-build/roborazzi-core/build.gradle
+++ b/include-build/roborazzi-core/build.gradle
@@ -56,7 +56,7 @@ kotlin {
 
     androidMain {
       dependencies {
-        def composeBom = platform('androidx.compose:compose-bom:2022.12.00')
+        def composeBom = platform(libs.androidx.compose.bom)
         implementation composeBom
         compileOnly libs.robolectric
         compileOnly libs.androidx.compose.ui.test
```

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'roborazzi'.
> Could not determine the dependencies of null.
   > Could not resolve all dependencies for configuration ':classpath'.
      > A problem occurred configuring project ':include-build:roborazzi-core'.
         > Failed to notify project evaluation listener.
            > compileSdkVersion is not specified. Please add it to build.gradle
            > Kotlin target 'android' tried to set up publishing for Android build variants that are not library variants or do not exist:
              * release
              Check the 'publishLibraryVariants' property, it should point to existing Android library variants. Publishing of application and test variants is not supported.
            > Kotlin target 'android' tried to set up publishing for Android build variants that are not library variants or do not exist:
              * release
              Check the 'publishLibraryVariants' property, it should point to existing Android library variants. Publishing of application and test variants is not supported.
            > Kotlin target 'android' tried to set up publishing for Android build variants that are not library variants or do not exist:
              * release
              Check the 'publishLibraryVariants' property, it should point to existing Android library variants. Publishing of application and test variants is not supported.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 115ms
```